### DISCUSSION TITLE
Scheduled user deletion via Offboarding Wizzard not working.

### DISCUSSION THREAD

**Discussion by MarcMast** [KelvinTegelaar/CIPP#2984] [Q&A]
I have an issue where the scheduled user deletion via Offboarding Wizzard  is not working.
Any other tasks specified in the Offboarding Wizzard goes through just fine but user deletion is not actioned (as verified when logged in to the 365 Admin portal for the tenant).
I am not getting any errors in CIPP and in  CIPP logs scheduled task shows as completed but the user is not deleted.
I am able to delete users via Identity Management menu with no issue. The problem manifests itself only with scheduled deletion.
I tested that with a couple of tenants.

Has anyone else run into this issue? If someone can confirm that this happens to them as well then perhaps I can raise it as a bug? Or alternatively if someone can advise that they were successfully able to schedule user deletion that would be helpful as well.

**Comment by KelvinTegelaar** [comment]
Hello,

Thank you for reaching out! This report has been classified as a support request rather than a bug report. To keep our development process focused, support requests are limited to paying users. This policy allows us to prioritize resources for those actively supporting CIPP, helping us maintain high-quality development and support.

Sponsors can contact our helpdesk directly via email for assistance with any issues or questions. For non-sponsor support, please refer to our documentation and community discussions, where you may find answers to similar queries.

Did you get this notification in error? Reply with a screenshot of your sponsorship payment, and we’ll gladly reopen the request.

Thank you for your understanding,
The CIPP Team
